tools/combine-files/combine_files.py

```
@@ -1311,6 +1312,270 @@ def _get_relative_path_or_original(config_file, file_path_root):
+            # No commits yet or detached HEAD
+            commit = "unknown"
+
+        # Determine the specific status of the file
+        file_path_normalized, status = _get_git_status(repo, file_path_in_repo)
+
+        return {
+            "repo_name": repo_name,
+            "file_path_in_repo": file_path_normalized,
+            "commit": commit,
+            "status": status,
+        }
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)